client/client.go

```diff
@@ -133,7 +133,7 @@ type jsonWriter interface {
 	WriteJSON(v interface{}) error
 }
 
-// New returns a new instance of Client
+// New returns a new Client with the given configuration.
```

Comment:
This was already there, but also a good example of unnecessary commenting.

I'll stop mentioning such cases now as the idea should be clear, but there are other instances.